src/ert/gui/model/snapshot.py

```diff
@@ -177,17 +172,8 @@ def _add_partial_snapshot(self, partial: PartialSnapshot, iter_: int) -> None:
         with ExitStack() as stack:
             iter_node = self.root.children[iter_]
             iter_index = self.index(iter_node.row(), 0, QModelIndex())
-            iter_index_bottom_right = self.index(
-                iter_node.row(), iter_index.column(), QModelIndex()
-            )
-            stack.callback(self.dataChanged.emit, iter_index, iter_index_bottom_right)
-
             reals_changed: List[int] = []
 
-            for idx in job_infos:
-                real_id = idx[0]
-                if real_id not in reals:
-                    reals[real_id] = RealizationSnapshot()
```

Comment:
Was this loop causing us to count every forward model twice? Once from snapshot.get_all_forward_models() and second from snapshot.reals?

Reply:
It is a possibility. I see no use in having this currently, so we decided to remove this code snippet.